pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/UserUtil.java

```diff
@@ -268,6 +271,29 @@ public static boolean checkUserAccess(String realm, String userId, String role,
         return false;
     }
 
+    public static boolean checkUserAccessForPatch(String realm, String userId, String role, String userIdRequest, Patch patch) {
+        String userEmailOrIdInPatch = patch.getUser();
+        String userIdFromPatch;
```

Comment:
I believe userId that is retrieved from the token is String, so we either have to convert one to String or the other one to int

Reply:
Right.  The token value is a String, but we're validating on the database value (as we should!) since that's actually the control.  I'd prefer to validate on that value but see what you mean in how we're coercing the USER_ID into and out of the token with String.